.mocharc.json

```diff
@@ -1,10 +1,10 @@
 {
   "diff": true,
-  "extension": "spec.ts",
+  "extension": ".ts",
   "package": "./package.json",
   "recursive": true,
   "reporter": "spec",
   "require": ["choma", "ts-node/register"],
-  "files": "test/**/*.spec.ts",
-  "watch-files": "test/**/*.spec.ts"
+  "files": "test/**/*.*.ts",
```

Comment:
Why was `spec` removed?

Reply:
Because I added Unit test case. File name is `utility.test.js`. But I agree to not use `*`, I added specific type `test/**/*.[spec|test].ts`